cmd/ocm/create/cluster/cmd.go

```diff
@@ -613,9 +622,7 @@ func promptClusterWideProxy() error {
 				}
 				args.existingVPC.Enabled = true
 			}
-		}
 
-		if args.clusterWideProxy.Enabled {
```

Comment:
why are we removing the check here?

Reply:
we just moving it.

Reply:
https://github.com/openshift-online/ocm-cli/pull/404/files#diff-2f19822c49dc0bec88a73920ea0f04a30b7047b5a827c90447e02b7378744f97R593
There was another one existed already